drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -96,6 +96,37 @@ static void syscmd_status_not(SHCI_TL_CmdStatus_t status)
 	BT_DBG("status:%d", status);
 }
 
+static void tryfix_event(TL_Evt_t *tev)
+{
+/*
+ * https://github.com/zephyrproject-rtos/zephyr/issues/19509
```

Comment:
Normally code comments use the same indentation level as the code itself, meaning this comment block should be indented by one extra tab.

Reply:
Alternatively, move the block right above the function (probably simpler since you've already got the text pre-formatted for the the full width)

Reply:
Moved right above the function. Thanks.